src/Enclosure.php

```diff
@@ -1115,7 +1115,7 @@ public function get_real_type($find_handler = false)
                     $type = 'audio/x-ms-wma';
                     break;
 
-                // Video mime-types
+                    // Video mime-types
```

Comment:
Weird indent, php-cs-fixer bug?

Reply:
Yes, this is a bug in php-cs-fixer. I move the comment under the case statement.